doc/api/tls.md

```diff
@@ -1090,9 +1090,9 @@ certificate.
 #### Certificate object
 <!-- YAML
 changes:
-  - version: v11.4.0
-    pr-url: https://github.com/nodejs/node/pull/24358
-    description: Support Elliptic Curve public key info.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/39809
+    description: Add fingerprint512
```

Comment:
Sorry, it seems that your commit replaced the existing change history instead of adding to the list.

I can't seem to get a GH suggestion to work here. The block should look like this:

```yaml
changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/39809
    description: Add fingerprint512
  - version: v11.4.0
    pr-url: https://github.com/nodejs/node/pull/24358
    description: Support Elliptic Curve public key info.
```

Reply:
@tniessen Derp, sorry about that! Fixed.